GettingStartedGuide.md

```diff
@@ -44,20 +44,25 @@ Once completed, one can progress to the
 ### 1.1 Hardware Requirements
 
 - Micro USB cable.
-- ESP32-C3, ESP32-C3, or a, ESP32-S3 board (e.g [ESP32-C3-DevKitC-02](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/hw-reference/esp32c3/user-guide-devkitc-02.html)).
+- ESP32-C3, ESP32-C3, or a, ESP32-S3 board (e.g
+[ESP32-C3-DevKitC-02](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/hw-reference/esp32c3/user-guide-devkitc-02.html)).
 - Personal Computer with Linux, MacOS, or Windows.
 - WiFi access point with access to the internet.
 
 ### 1.2 Software Requirements
 
-- [ESP-IDF 5.0](https://github.com/espressif/esp-idf/tree/v5.0) or higher to
-configure, build, and flash the project. To setup for the ESP32-C3,
-follow Espressif's
+- [ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started):
+This is used to configure, build, and flash the project.
+To setup for the ESP32-C3, follow Espressif's
 [Getting Started Guide for the ESP32-C3](https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/get-started/index.html).
 
-    **NOTE:** As of December 15h 2023 ESP-IDF V4.4.3 is no longer supported. For
-    more information refer to the
-    [ESP-IDF Release Support Schedule](https://github.com/espressif/esp-idf#esp-idf-release-support-schedule)
+  - The versions of `ESP-IDF` supported by this repository are the ones supported by
+    [Espressif's GitHub Build Action](https://github.com/espressif/esp-idf-ci-action?tab=readme-ov-file)
+  - To see the currently supported `ESP-IDF` versions please refer to the list in
+    [build.yml](./.github/workflows/build.yml#L22).
+
+    **NOTE:** Please do not submit a bug report due to build errors or demo failures
+    when using an unsupported version of `ESP-IDF`.
```

Comment:
I like this. One question - do you think we should make it clear that 'released' versions are the supported versions and that 'development' branches may not work?

Reply:
You're right, we absolutely should if we're going to add a line like this.
I'll add a link to it, good idea!